File: RuntimeSystem-Benchmarking/Code/Simple Obj Python/parent.py
class parent():
    first = None
    sec = 4
    # The constructor for instance variables
    def __init__(self, third, forth):
        self.third = third
        self.forth = forth

    # This is the static method
    #The @staticmethod here is the decorator and not necessary
    # Since the function does not get any obj of class(self) as the frist
    # argument, it is automatically static
    #@statistics
    def rangeFunc(x):
        if len(x) < 9:
            print("Not in the range")
            return None
        return x[3,9]
    # This is the instance method
    # The @classmethod here is the decorator and not necessary
    # Since the function does gets an obj of class(self) as the frist
    # argument, it is automatically instance method
    #@classmethod
    def multiplier(self, x):
        print(x * self.third)

File: RuntimeSystem-Benchmarking/Code/Simple Obj Java/parent.java

import java.util.Arrays;
public class parent {
    static int first;
    static int sec = 4;
    double third = 10;
    double forth;

    // The constructor for instance variables
    public parent(double third, double forth){
        this.third = third;
        this.forth = forth;
    }
    public static int[] rangeFunc(int[] x ){
        if(x.length < 10){
            System.out.println("Not in the range");
            return null ;
        }
        return Arrays.copyOfRange(x,3,9);
    }
    public void multiplyer(int x ){
        System.out.println(x * third);
    }
}
